spring-petclinic/pom.xml

```diff
@@ -626,10 +352,7 @@
 		</dependency>
 
 		<!-- For MySql only -->
-		<!-- <dependency>
-			<groupId>mysql</groupId>
-			<artifactId>mysql-connector-java</artifactId>
-		</dependency> -->
+		<!-- <dependency> <groupId>mysql</groupId> <artifactId>mysql-connector-java</artifactId> </dependency> -->
```

Comment:
In case of mysql I think it will be handled by EAP7 and not by the application. The application doesn't deal directly with MySQL. So I assume it's safe to remove the mysql stuff from the pom.xml